features/guidance/anticipate-lanes.feature

```diff
@@ -259,6 +259,128 @@ Feature: Turn Lane Guidance
+
+    @anticipate-roundabout
+    Scenario: Roundabout with lanes only tagged on exit
+        Given the node map
+            |   |   | e |   |   |
+            | a | b |   | d | f |
+            |   |   | c |   |   |
+
+        And the ways
+            | nodes | turn:lanes:forward                    | highway | junction   |
+            | ab    |                                       | primary |            |
+            | bc    |                                       | primary | roundabout |
+            | cd    | slight_left\|slight_left&slight_right | primary | roundabout |
+            | de    |                                       | primary | roundabout |
+            | eb    |                                       | primary | roundabout |
+            | df    |                                       | primary |            |
+
+        When I route I should get
+            | waypoints | route    | turns                                                 | lanes                                                    |
+            | a,f       | ab,df,df | depart,roundabout-exit-1,use lane slight right,arrive | ,,slight left:false slight left:false slight right:true, |
+
+    @anticipate-roundabout
+    Scenario: Anticipate with lanes in roundabout where we stay on the roundabout for multiple exits
+        Given the node map
+            |   |   | a |   |   |
+            |   |   | b |   |   |
+            |   | c |   | g | h |
+            |   |   |   |   |   |
+            |   | d |   | f |   |
+            |   |   | e |   |   |
+            | x |   |   |   | y |
+
+        And the ways
+            | nodes | turn:lanes:forward         | highway | junction   |
+            | ab    | slight_right\|slight_right | primary |            |
+            | bc    |                            | primary | roundabout |
+            | cd    |                            | primary | roundabout |
+            | de    |                            | primary | roundabout |
+            | ef    |                            | primary | roundabout |
+            | fg    | slight_right               | primary | roundabout |
+            | gb    |                            | primary | roundabout |
+            | gh    |                            | primary |            |
+            | cx    |                            | primary |            |
+            | dx    |                            | primary |            |
+            | ey    |                            | primary |            |
+            | fy    |                            | primary |            |
+
+        When I route I should get
+            | waypoints | route    | turns                           | lanes                                  |
+            | a,h       | ab,gh,gh | depart,roundabout-exit-5,arrive | ,slight right:false slight right:true, |
+
+    @anticipate-roundabout-onion
+    Scenario: Anticipation in Onion Roundabout spiraling inwards / outwards
```

Comment:
as discussed in https://github.com/Project-OSRM/osrm-backend/issues/2621, this is currently not a valid roundabout model for OSRM and cannot be handled correctly.